deprecations/README.md

```diff
@@ -36,6 +36,7 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `CustomerId`<br>in [Add order](../operations/orders.md#add-order) | Replaced by `AccountId` | ?? Jan 2023 | - |
```

Comment:
Let's don't forget about discontinued date, I think this is a minor change so half year should be fine.